.gitignore

```diff
@@ -62,12 +62,14 @@ install_manifest.txt
 # vim temp files
 *.swp
 *.un~
+*.*~
 
 # Emacs files:
 # VCS comparison files:
 *.~*~
 # Autosave files
 .#*
+#*#
```

Comment:
[#]*[#]
It should have been this format, to ignore temporary files with this name #filename#